app/views/people/_current_roles.html.erb

```diff
@@ -1,5 +1,5 @@
 <% if person.currently_in_a_role? %>
-  <section id="current-roles" dir="<%= page_text_direction %>" lang="<%= person.locale %>">
+  <section id="current-roles" class="<%= direction_rtl_class %>" <%= dir_attribute %> <%= lang_attribute %>>
```

Comment:
Will this output `class=""` if `page_text_direction` set to `ltr`?

Reply:
ah, it will. good spot.
We could do something like:
`<%= "class=#{direction_rtl_class}" if direction_rtl_class %>` but that is not particularly tidy